proto/cosmos/accesscontrol_x/genesis.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package cosmos.accesscontrol_genesis.v1beta1;
+package cosmos.accesscontrol_x.v1beta1;
```

Comment:
Renaming the package since we need the query, gov and genesis protos to be in the /x/ module to avoid circular imports